angular.json

```diff
@@ -28,9 +24,13 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "./node_modules/bootstrap/dist/css/bootstrap.css",
+              "src/styles.css"
```

Comment:
I can't find any styles.css inside project repo, remove it if unnecessary or change it to styles.scss

Reply:
Removed unused path